app/components/FileRow.js

```diff
@@ -214,9 +214,7 @@ export default class FileRow extends Component {
 
     const file = this.props.file || {};
 
-    const metadata = file.game.getMetadata() || {};
-    const startAt = metadata.startAt;
-    const startAtDisplay = timeUtils.convertToDateAndTime(startAt) || 'Unknown';
+    const startAtDisplay = timeUtils.fileToDateAndTime(file.game, file.fileName, file.fullPath) || "Unknown";
```

Comment:
Why not get rid of the format arg, always return a moment, and then just use `file.startTime.format('ll · LT')`, making sure that the time is not null, of course.

Reply:
I wanted to avoid using moment within places other than timeUtils, especially because app/components/stats/GameProfile.js also uses convertToDateAndTime. If you're ok with using moment in other places then I can change it